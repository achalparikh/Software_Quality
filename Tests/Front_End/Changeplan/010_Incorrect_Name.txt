login
admin
changeplan
Jeff Doe
00012
logout
quit