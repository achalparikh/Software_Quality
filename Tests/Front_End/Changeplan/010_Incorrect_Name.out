Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as "John Doe"
Enter a command:
Enter an account holder:
Error, user Jeff Doe does not match any accounts.
Enter a command:
Error, invalid command, please enter a command:
You are now logged out
Login to start:
